test/command/suite/io_flush/recursive/dependent/table/create.test

```diff
@@ -0,0 +1,11 @@
+table_create Users TABLE_HASH_KEY ShortText
+
+#@collect-query-log true
+io_flush --recursive dependent
+#@collect-query-log false
```

Comment:
What is tested here?

Reply:
@kou I want to test before/after `table_create`, how io_flush output is changed.